### DISCUSSION TITLE
Cloud Tier - Record Limit

### DISCUSSION THREAD

**Discussion by scotttiey** [Budibase/budibase#7078] [General]
Hi I’m looking to build an app for a customer but dont want to self host but will need more than a 1,000 records. Does the cloud tier have the option to support more than a 1,000 records? Understand this may involve pricing

**Comment by joebudi** [comment]
Hi @scotttiey . We're working on releasing our paid tiers over the new couple of weeks. The paid tiers will lift the record limit.

**Reply by andyburgessmd** [reply]
What happens at the moment if the limit on records or automations is hit? Will any new records or triggers fail?
And is the limit on automations per time period (month)?

**Comment by scotttiey** [comment]
thanks @joebudi look forward to it!
